sample_data/README.md

```diff
@@ -4,14 +4,33 @@ We can use ClickHouse online service with full data access to make ClikcHouse sa
 
 ## Usage
 
+### ClickHouse server image
+
+- x86: `docker pull docker-hub.x-lab.info/opendigger/open-digger-clickhouse-base:v1`
+- ARM: `docker pull docker-hub.x-lab.info/opendigger/open-digger-clickhouse-base-arm:v1`
+
 ### Use sample data
 
 To use sample data from OSS service, you need to follow the steps:
 
-- Download data from OSS. We provide several sample datasets in the table below. 
-- Extract data from archive file to a folder: `tar -zxvf data.tar.gz -C ./folder_path`. You will get a `table` and a `data` file.
-- Use ClickHouse base image with extracted data to initialize the database. The extracted `data` and `table` file should be mounted into `/data/` folder into the container, here is an example: `docker run -d --name container_name -m 6G -p 8123:8123 -p 9000:9000 --ulimit nofile=262144:262144 --volume=./folder_path/:/data/ open-digger-clickhouse-base:v1`
-- The data is ready until message `Insert data done.` logged into container console. Now the Clickhouse container is running. Stop and restart the same container instance will not import data again.
+1. Download data from OSS. We provide several sample datasets in the table below. 
+
+2. Extract data from archive file to a folder: `tar -zxvf data.tar.gz -C ./folder_name`. You will get a `table` and a `data` file.
+
+3. Use ClickHouse base image with extracted data to initialize the database. The extracted `data` and `table` file should be mounted into `/data/` folder into the container, here is an example:
+   - Linux: `docker run -d --name container_name -m 6G -p 8123:8123 -p 9000:9000 --ulimit nofile=262144:262144 --volume=$(pwd)/folder_name/:/data/ open-digger-clickhouse-base:v1`;
```

Comment:
May change `Linux` to `Linux/MacOS` here.

Reply:
Agree. Will do it right now.